styles/widgets/common/button.less

```diff
@@ -62,6 +62,7 @@
     width: 0;
     font-size: 0;
     opacity: 0;
+    display: none;
```

Comment:
@LazyLahtak

Reply:
I think that we can get rid of these rules.

Reply:
Are you sure? :)

Reply:
In renovated button, when we use template, we render `submit input` before `content`. In old button we always render this input after `content`.
Without `display: none` it may cause some visual errors and difference between old and new button.

I saved these rules to avoid any issues, but I've rechecked and seems that with `display: none` we have no any reason to save other rules.

Reply:
/cc @DokaRus